packages/terra-action-header/src/ActionHeader.jsx

```diff
@@ -88,7 +87,14 @@ const ActionHeader = ({
     ? (
       <FormattedMessage id="Terra.actionHeader.close">
         {closeText => (
-          <Button className={cx('header-close-button')} isIconOnly icon={<span className={cx(['header-icon', 'close'])} />} text={closeText} onClick={onClose} />
+          <Button
+            className={cx('header-close-button')}
+            isIconOnly
+            icon={<span className={cx(['header-icon', 'close'])} />}
+            text={closeText}
+            onClick={onClose}
+            variant={Button.Opts.Variants.UTILITY}
```

Comment:
can we pass a string or use ButtonVariants instead of opts variants?

Reply:
We can but what is the trade-off here when `Button` needs to be imported here irregardless?

Reply:
Less properties to access. `'utility'` or `Button.UTILITY` is cleaner and (probably) slightly more performant than `Button.Opts.Variants.UTILITY`

Reply:
Alright, here you go: https://github.com/cerner/terra-core/pull/2714/commits/f7b588961957cd4c39c4a1e27071a6ac6e9924fc

This usage is not consistent with the how the variants are accessed across our code, tho.

Reply:
Opts was a first pass creation. We've had discussions about moving away from this pattern but they still exist because of passivity. https://github.com/cerner/terra-core/issues/1044